setup.py

```diff
@@ -1,22 +1,28 @@
+import ast
 import os
-import re
 import codecs
 from setuptools import setup, find_packages
 
 
+class VersionFinder(ast.NodeVisitor):
+    def __init__(self):
+        self.version = None
+
+    def visit_Assign(self, node):
+        if node.targets[0].id == '__version__':
+            self.version = node.value.s
+
+
 def read(*parts):
     filename = os.path.join(os.path.dirname(__file__), *parts)
     with codecs.open(filename, encoding='utf-8') as fp:
         return fp.read()
 
 
-def find_version(*file_paths):
-    version_file = read(*file_paths)
-    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
-                              version_file, re.M)
-    if version_match:
-        return version_match.group(1)
-    raise RuntimeError("Unable to find version string.")
+def find_version(*parts):
```

Comment:
I'm not sure I understand the reason for this. Why not simply `__import__('dbtemplates').version`?

(Works fine for me. This is only a question.)